addons/web/static/src/views/fields/badge_selection/badge_selection_field.js

```diff
@@ -16,10 +17,12 @@ export class BadgeSelectionField extends Component {
             validate: (s) => ["sm", "md", "lg"].includes(s),
             default: "md",
         },
+        colorField: { type: String, optional: true },
     };
 
     setup() {
         this.type = this.props.record.fields[this.props.name].type;
+        this.isDarkMode = cookie.get("color_scheme");
```

Comment:
nitpicking: not necessary to store on `this`: you can directly read the cookie below

Also: the variable name is pretty bad. I expected the variable to be boolean